src/script/cpp_api/s_base.cpp

```diff
@@ -153,13 +153,6 @@ ScriptApiBase::ScriptApiBase(ScriptingType type):
 	lua_pushstring(m_luastack, porting::getPlatformName());
 	lua_setglobal(m_luastack, "PLATFORM");
 
-#ifdef HAVE_TOUCHSCREENGUI
```

Comment:
Should I go back for LUA to something similar too?

Reply:
I'm not sure what you mean. Your current approach looks good.

If you mean going back to the "check for `g_touchscreengui`" approach: That's not possible since we're talking about mainmenu Lua code, where `g_touchscreengui` is always null.